client/my-sites/customer-home/cards/primary/quick-links/index.jsx

```diff
@@ -160,15 +160,17 @@ export const QuickLinks = ( {
 	if ( ! isMobile() ) {
 		return (
 			<Card className="quick-links">
-				<CardHeading>{ translate( 'Quick Links' ) }</CardHeading>
+				<CardHeading tagName="h2" size={ 16 }>
+					{ translate( 'Quick links' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **47** times:
`translate( 'Quick Links' )` *ES Score: 10*